docs/blog/2020-07-13-i18n-pain-points/index.md

```diff
@@ -9,7 +9,7 @@ tags:
 
 During Gatsby Days Reconfigured last month [we previewed quite a few exciting projects](/blog/2020-06-23-Reconfiguring-Gatsby-Days/) to be released soon. These launches are aimed at helping developers transition their CMSs into the headless CMS era; enabling content creators to use Gatsby more easily and intuitively; and, simultaneously, improving the developer experience for our current and future users. One of these exciting new announcements actually ties all three of these goals together: a sneak peek at the development of our own [i18n theme](/blog/2020-06-23-Reconfiguring-Gatsby-Days/#i18n-theme). It's the first step towards having internationalization in Gatsby as a first-class citizen, and we are looking to launch by the end of the month.
 
-First, though, we’d like to share the research process we’ve followed in developing the i18n theme, and the pain points we sought to solve. The goal in sharing these findings is to start an open discussion and invite your feedback. If you’re interested in contributing, please join the [Gatsby Discord channel](https://discord.gg/cQ2MPUz) and add your voice to the conversation!
+First, though, we’d like to share the research process we’ve followed in developing the i18n theme, and the pain points we sought to solve. The goal in sharing these findings is to start an open discussion and invite your feedback. If you’re interested in contributing, please join the [Gatsby Discord channel](https://gatsby.dev/discord) and add your voice to the conversation!
```

Comment:
Please revert the discord changes, this is a specific invite to a channel and thus the general link is not the correct choice.